core/src/main/java/org/apache/accumulo/core/conf/ClientConfigGenerate.java

```diff
@@ -52,7 +52,7 @@ void generate() {
       generateSection("Scanner", "scanner.");
       generateSection("SSL", "ssl.");
       generateSection("SASL", "sasl.");
-      generateSection("Tracing", "general.opentelemetry.");
+      generateSection("Tracing", "trace.");
```

Comment:
This change is just a simple revert to what was there before. It will preserve the deprecated client trace properties in the docs.